src/app/pages/send-tokens/hooks/use-selected-asset.ts

```diff
@@ -6,7 +6,7 @@ import { ftDecimals, stacksValue } from '@app/common/stacks-utils';
 import { useCurrentAccountAvailableStxBalance } from '@app/query/balance/balance.hooks';
 import { useSelectedAssetMetadata } from '@app/store/assets/asset.hooks';
 
-export function useSelectedAsset(assetId: string) {
+export function useSelectedAsset(assetId?: string) {
```

Comment:
When arguments are made optional, the responsibility of whether or not the data is present moves from the invoker to the function itself. This isn't an improvement imo.

There's no legit case where I'd want to view details of no asset. The only reason we do this is because, in some cases, the invoking code doesn't have an `assetId` yet. That's a problem for the component using this, and not the hook itself.

Reply:
I will just change all these back, np, it was a pretty minor part of this PR ...not really the refactor. I was just addressing an error showing for me in `transaction-request`.